cob_bringup/robots/cob3-2.xml

```diff
@@ -0,0 +1,177 @@
+			<arg name="component_name" value="tray" />
+			<arg name="robot" value="$(arg robot)" />
+		</include>
+		<include file="$(find cob_bringup)/controllers/cob_trajectory_controller.launch">
+			<arg name="robot" value="$(arg robot)"/>
+			<arg name="component_name" value="tray"/>
+		</include>
+		<include file="$(find cob_bringup)/drivers/schunk_powercube_chain_driver.launch" >
+			<arg name="component_name" value="arm" />
+			<arg name="robot" value="$(arg robot)" />
+		</include>
+		<include file="$(find cob_bringup)/controllers/cob_trajectory_controller.launch">
+			<arg name="robot" value="$(arg robot)"/>
+			<arg name="component_name" value="arm"/>
+		</include>
+		<include file="$(find cob_bringup)/drivers/schunk_sdh.launch" >
+			<arg name="robot" value="$(arg robot)" />
+		</include>
+
+		<include file="$(find cob_bringup)/drivers/head_axis.launch" >
+			<arg name="robot" value="$(arg robot)" />
+		</include>
+
+		<!-- start additional packages -->
+		<include file="$(find cob_bringup)/tools/diagnostics_aggregator.launch" >
+			<arg name="robot" value="$(arg robot)" />
+		</include>
+		<include file="$(find cob_bringup)/tools/teleop.launch" >
+			<arg name="robot" value="$(arg robot)" />
+			<arg name="version" value="v1" />
+		</include>
+		<include file="$(find cob_bringup)/drivers/openni2.launch" >
+			<arg name="robot" value="$(arg robot)" />
+			<arg name="name" value="cam3d"/>
+			<arg name="device_id" value="#1"/>
+		</include>
+		<!--include file="$(find cob_bringup)/drivers/image_flip_nodelet.launch" >
+			<arg name="robot" value="$(arg robot)" />
+			<arg name="nodelet_manager" value="cam3d_nodelet_manager"/>
+			<arg name="start_manager" value="false"/>
+		</include-->
+		<include file="$(find cob_cam3d_throttle)/launch/cam3d_throttle.launch" >
+			<arg name="namespace" value="/cam3d" />
+			<arg name="rate" value="1.0" />
+			<arg name="start_manager" value="false"/>
+			<arg name="nodelet_manager" value="cam3d_nodelet_manager"/>
+		</include>
+
+		<!-- aggregated robot_state_publisher -->
+		<include file="$(find cob_bringup)/tools/robot_state_publisher.launch">
+			<arg name="robot" value="$(arg robot)" />
+		</include>
+	</group>
+
+	<group>
+		<machine name="pc2" address="$(arg pc2)" env-loader="$(find cob_bringup)/env.sh" default="true" timeout="30"/>
+
+		<!-- pc monitor -->
+		<include file="$(find cob_bringup)/tools/pc_monitor.launch" >
+			<arg name="robot" value="$(arg robot)" />
+			<arg name="pc" value="$(arg pc2)" />
+		</include>
+
+		<!-- start camera sensors -->
+		<include file="$(find cob_bringup)/drivers/prosilica.launch" >
+			<arg name="robot" value="$(arg robot)" />
+			<arg name="name" value="right"/>
+		</include>
+		<include file="$(find cob_bringup)/drivers/prosilica.launch" >
+			<arg name="robot" value="$(arg robot)" />
+			<arg name="name" value="left"/>
+		</include>
+
+		<include file="$(find cob_bringup)/drivers/dsa.launch" >
+			<arg name="robot" value="$(arg robot)" />
+		</include>
```

Comment:
dsa.launch the tactile sensor in the sdh, right? then it should go together with schunk_sdh.launch, i.e. pc1?